Index: contrib/libamrwb/configure.ac
===================================================================
--- contrib/libamrwb/configure.ac	(revision 10951)
+++ contrib/libamrwb/configure.ac	(working copy)
@@ -43,10 +43,10 @@
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
-AC_TYPE_INT16_T
-AC_TYPE_INT32_T
-AC_TYPE_INT8_T
-AC_TYPE_UINT16_T
+#AC_TYPE_INT16_T
+#AC_TYPE_INT32_T
+#AC_TYPE_INT8_T
+#AC_TYPE_UINT16_T
 
 # Checks for library functions.
 AC_FUNC_MALLOC
Index: contrib/libamrnb/configure.ac
===================================================================
--- contrib/libamrnb/configure.ac	(revision 10951)
+++ contrib/libamrnb/configure.ac	(working copy)
@@ -43,10 +43,10 @@
 
 # Checks for typedefs, structures, and compiler characteristics.
 AC_C_CONST
-AC_TYPE_INT16_T
-AC_TYPE_INT32_T
-AC_TYPE_INT8_T
-AC_TYPE_UINT8_T
+#AC_TYPE_INT16_T
+#AC_TYPE_INT32_T
+#AC_TYPE_INT8_T
+#AC_TYPE_UINT8_T
 AC_C_VOLATILE
 
 # Checks for library functions.
Index: configure.ac
===================================================================
--- configure.ac	(revision 10951)
+++ configure.ac	(working copy)
@@ -1,7 +1,7 @@
 #                                               -*- Autoconf -*-
 # Process this file with autoconf to produce a configure script.
 
-AC_PREREQ([2.60])
+AC_PREREQ([2.59])
 AC_INIT(sipXmediaLib, 3.3, sipxtapi-dev@list.sipfoundry.org)
 AC_CONFIG_AUX_DIR(config)
 AC_PREFIX_DEFAULT([/usr/local/sipx])
Index: contrib/libspeex/doc/Makefile.am
===================================================================
--- contrib/libspeex/doc/Makefile.am	(revision 13928)
+++ contrib/libspeex/doc/Makefile.am	(working copy)
@@ -1,3 +1,5 @@
+docdir = ${datadir}/doc/${PACKAGE}
+
 doc_DATA = manual.pdf
 
 EXTRA_DIST = $(doc_DATA)
