--- w3c-libwww.spec.orig	2006-07-31 21:08:13.000000000 -0400
+++ w3c-libwww.spec	2006-07-31 21:21:11.000000000 -0400
@@ -13,6 +13,7 @@
 URL: http://www.w3.org/Library
 BuildRoot: /var/tmp/%{name}-root
 BuildRequires: libtool autoconf automake zlib-devel
+Provides: sipx-w3c-libwww
 
 %description
 Libwww is a general-purpose Web API written in C for Unix and Windows (Win32).
@@ -24,7 +25,7 @@
 %package devel
 Summary: Libraries and header files for programs that use libwww.
 Group: Development/Libraries
-Requires: w3c-libwww = %{version} zlib-devel
+Requires: sipx-w3c-libwww = %{version} zlib-devel
 
 %description devel
 Static libraries and header files for libwww, which are available as public
@@ -33,7 +34,7 @@
 %package apps
 Summary: Applications built using Libwww web library: e.g. Robot, command line tool, etc.
 Group: Applications/Internet
-Requires: w3c-libwww
+Requires: sipx-w3c-libwww
 
 %description apps
 
@@ -69,7 +70,7 @@
 #CFLAGS="$RPM_OPT_FLAGS" ./configure --prefix=/usr --with-regex --with-zlib
 #make RPM_OPT_FLAGS="$RPM_OPT_FLAGS"
 
-%configure --enable-shared --with-gnu-ld --with-regex --with-zlib
+%configure --enable-shared --with-gnu-ld --with-regex --with-zlib --with-ssl
 export tagname=CC
 make LIBTOOL=/usr/bin/libtool
 
